app/assets/scss/_iframe_preview.scss

```diff
@@ -31,3 +31,10 @@ $app-mobile-border-bottom-width: 68px;
   height: 1000px;
   border: 0;
 }
+
+.draft-preview {
+  background-image: file-url("draft-watermark.png");
```

Comment:
This could have been a SVG or better yet it could have been text in a div and using CSS to rotate the text.  Both options could have scaled based on the size of the viewing point so it didn't pixelate.

Reply:
I copied it directly from the GOV.UK assets...